setup.py

```diff
@@ -39,39 +37,41 @@
     author='SearXNG dev team',
     author_email='contact@searxng.org',
     license='GNU Affero General Public License',
-    packages=find_packages(exclude=["tests*", "searxng_extra"]),
-    zip_safe=False,
+    packages=find_packages(
+        include=[
+            'searx', 'searx.*', 'searx.*.*', 'searx.*.*.*',
+        ]
+    ),
     install_requires=requirements,
     extras_require={
         'test': dev_requirements
     },
     entry_points={
         'console_scripts': [
-            'searx-run = searx.webapp:run',
-            'searx-checker = searx.search.checker.__main__:main'
+            'searxng-run = searx.webapp:run',
+            'searxng-checker = searx.search.checker.__main__:main'
         ]
     },
     package_data={
         'searx': [
             'settings.yml',
-            '../README.rst',
-            '../requirements.txt',
-            '../requirements-dev.txt',
-            'data/*',
-            'info/*',
-            'info/*/*',
-            'plugins/*/*',
-            'static/*.*',
-            'static/*/*.*',
-            'static/*/*/*.*',
-            'static/*/*/*/*.*',
-            'static/*/*/*/*/*.*',
-            'templates/*/*.*',
-            'templates/*/*/*.*',
-            'tests/*',
-            'tests/*/*',
-            'tests/*/*/*',
-            'translations/*/*/*'
+            '*.toml',
+            '*.msg',
+            'search/checker/scheduler.lua',
+            'data/*.json',
+            'data/*.txt',
+            'data/*.ftz',
+            'infopage/*/*',
+            'static/themes/simple/css/*',
+            'static/themes/simple/css/*/*',
+            'static/themes/simple/img/*',
+            'static/themes/simple/js/*',
+            'templates/*/*',
+            'templates/*/*/*',
+            'translations/*',
+            'translations/*/*',
+            'translations/*/*/*',
+
```

Comment:
```suggestion
```

Reply:
Oops .. I tidied up the patch .. BTW I fixed the python version to py3.8 (py3.7 is not supported for a longer time) --> [diff](https://github.com/searxng/searxng/compare/1fc1d5fd76647dc0cc0ea10a71d2a991e6c04e96..8e67431cd86fa3922bdeca261d6721ecd592a501)